hugo/content/steps/issue/jira_edit_issue.md

```diff
@@ -15,6 +15,7 @@ Updates an existing issue based on given input, which should have some minimal i
 
 * **idOrKey** - issue id or key.
 * **issue** - issue to be updated.
+* **queryParams** - Optional. Map of query parameters. 
```

Comment:
Can you add an example here please, especially on how to send notifyUsers flag, since the initial request was to add support around this parameter?